src/gtk-3.20/scss/widgets/_choosers.scss

```diff
@@ -168,7 +168,10 @@
             &:backdrop { border-top-color: $backdrop_borders_color; }
         }
 
-        #pathbarbox { border-bottom: 1px solid $bg_color; }
+        #pathbarbox {
```

Comment:
and what's that?

Reply:
ah, i see your comment, are you sure it's not taking $menu_bg_color there?

Reply:
i took a look on the dialog: yup, it's good idea what you are proposing

Reply:
are you gonna backport this to older GTK+3? if not i am merging already

Reply:
I don't see it in older gtk+3. You can merge for me